plugins/AccountStats.py

```diff
@@ -118,36 +118,48 @@ def notify_stats(self):
             return
 
         self.earnings = {}
+        output = ''
 
         # Today's earnings
         cursor = self.db.execute(DB_GET_TODAYS_EARNINGS)
-        output = ''
-        if cursor.rowcount > 0:
-            for row in cursor:
+        row = cursor.fetchone()
+        if row is not None:
+            while row is not None:
                 output += self.format_value(row[0]) + ' ' + str(row[1]) + ' Today\n'
                 if row[1] not in self.earnings:
                     self.earnings[row[1]] = {}
                 self.earnings[row[1]]['todayEarnings'] = row[0]
+                row = cursor.fetchone()
         else:
             output += 'None Today\n'
         cursor.close()
 
```

Comment:
I don't like using fetchall() with any RDBMS because the number of rows returned is variable depending on the amount of data stored. In this case, we know it won't be 10,000 rows, but still, it is a better practice to loop rather than eat up memory. You are also, additionally, spending CPU time iterating through the array in order to count the number of elements where `if .. is not None` is a simple evaluation.